frontend/src/types/UserTypes.ts

```diff
@@ -1,5 +1,6 @@
-export type AdminUser = {
+export type User = {
```

Comment:
do you mind creating a `AdminUser` and `TeacherUser`? so we can enforce that teacher users have a required school field